mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -146,11 +152,21 @@ public static Response showMCRObject(String pathParamId, String queryParamStyle,
         StringWriter sw = new StringWriter();
         XMLOutputter outputter = new XMLOutputter(Format.getPrettyFormat());
         try {
-            outputter.output(doc, sw);
+            if (MCRConfiguration.instance().getString("MCR.ContentTransformer.RestApiXmlFilter.Stylesheet", "").length() > 0) {
+                MCRContentTransformer trans = MCRContentTransformerFactory.getTransformer("RestApiXmlFilter");
```

Comment:
use the value of `MCR.RestAPI.v1.Filter.XML` here and do not hard code a specific transformer here